pysmspp/data/blocks/UCBlock.csv

```diff
@@ -6,6 +6,9 @@ UCBlock,NumberUnits,Dimension,Dimension,int,-,-,,required,Number of units
 UCBlock,NumberElectricalGenerators,Dimension,Dimension,int,-,-,,optional,Number of electrical generators
 UCBlock,NumberNodes,Dimension,Dimension,int,-,-,1.0,optional,Number of nodes
 UCBlock,NumberLines,Dimension,Dimension,int,-,-,,optional,Number of lines
+UCBlock,NumberArcs,Dimension,Dimension,int,-,-,,optional,Number of arcs
+UCBlock,NumberReservoirs,Dimension,Dimension,int,-,-,,optional,Number of reservoirs
+UCBlock,TotalNumberPieces,Dimension,Dimension,int,-,-,,optional,Number of pieces of the power flow function
```

Comment:
Just to align, this makes these parameters general for all hydroblocks, correct?